<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>852</width>
    <height>574</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="bns1">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>300</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="num3">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>180</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="num1">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>180</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="num4">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>180</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="num6">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>180</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="potmoney">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>490</y>
     <width>421</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="data1">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>100</y>
     <width>191</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>통계</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="num5">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>180</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>270</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>직접 조회를 원하면</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="turn">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>480</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>1등 상금</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="num2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>180</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="calllottery">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>180</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>최신 회차 당첨번호 조회</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>290</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>아래에 입력하세요!</string>
   </property>
  </widget>
  <widget class="QLabel" name="bns2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>250</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>보너스 번호</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="turnnum">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>320</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="numbutton">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>320</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>조회</string>
   </property>
  </widget>
  <widget class="QLabel" name="sixnum">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>130</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>당첨 번호</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="date">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>30</y>
     <width>331</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>-20</x>
     <y>-50</y>
     <width>1071</width>
     <height>691</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>slotmachine.jpg</pixmap>
   </property>
  </widget>
  <zorder>label_2</zorder>
  <zorder>bns1</zorder>
  <zorder>num3</zorder>
  <zorder>num1</zorder>
  <zorder>num4</zorder>
  <zorder>num6</zorder>
  <zorder>potmoney</zorder>
  <zorder>data1</zorder>
  <zorder>num5</zorder>
  <zorder>label_3</zorder>
  <zorder>turn</zorder>
  <zorder>label</zorder>
  <zorder>num2</zorder>
  <zorder>calllottery</zorder>
  <zorder>label_4</zorder>
  <zorder>bns2</zorder>
  <zorder>turnnum</zorder>
  <zorder>numbutton</zorder>
  <zorder>sixnum</zorder>
  <zorder>date</zorder>
 </widget>
 <resources>
  <include location="test.qrc"/>
 </resources>
 <connections/>
</ui>
